src/app/data-model-provider/MetadataTypes.h

```diff
@@ -87,6 +87,7 @@ enum class CommandQualityFlags : uint32_t
 {
     kFabricScoped = 0x0001,
     kTimed        = 0x0002, // `T` quality on commands
+    kIsLarge      = 0x0003, // `L` quality on commands
```

Comment:
```suggestion
    kLargeMessage      = 0x0004, // `L` quality on commands
```
These are bitmask values, not bit offsets.  And the name of the quality in the spec is "Large Message", though maybe the flag should be called "Large Payload"...

The good news is that CI caught the functional bug, at least....